aspnetcore/fundamentals/minimal-apis/7.0-samples/todo-group/Program.cs

```diff
@@ -0,0 +1,201 @@
+using Data;
+using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+var connection = new SqliteConnection("DataSource=:memory:");
+connection.Open();
+// Add InMemoryDatabase to the container.
+builder.Services.AddDbContext<ApplicationDbContext>(options =>
+{
+    options.UseSqlite(connection);
+});
+
+var app = builder.Build();
+
+// Configure the HTTP request pipeline.
+if (app.Environment.IsDevelopment())
+{
+    // localhost:{port}/swagger
+    app.UseSwagger();
+    app.UseSwaggerUI();
+}
+
+app.MapGet("/", () => "Hello World!");
+
+// todo endpoints
+var todos = app.MapGroup("/todos").WithTags("Todo Endpoints");
+todos.MapGet("/", GetAllTodos);
+todos.MapGet("/{id}", GetTodo);
+todos.MapPost("/", CreateTodo).AddRouteHandlerFilter((context, next) =>
+{
+    if (context.HttpContext.Request.ContentLength == 0)
```

Comment:
Can this actually be hit? Unless `TodoDto` is made optional in the `CreateTodo` method signature by making it nullable, I think any 0-length request would be rejected before getting to this stage.

Also, `AddRouteHandlerFilter` will be renamed to `AddEndpointFilter` in preview7. I'm not sure how early we try to update that. https://github.com/dotnet/aspnetcore/issues/42589

Reply:
Ok, I would work on that. 
Also wanted to ask the same question in regards to the change